assets/js/hooks/useFeature.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Feature Flags hook.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { FeaturesContext } from '../components/FeaturesProvider/context';
+
+/**
+ * Returns the enabled state of a feature flag.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} feature The feature flag name to check enabled state for.
+ * @return {boolean} `true` if the feature is enabled, `false` otherwise.
+ */
+export const useFeature = ( feature ) => {
+	const enabledFeatures = useContext( FeaturesContext );
+
+	return !! get( enabledFeatures, feature, false );
```

Comment:
`get` isn't needed here as `enabledFeatures` should always be an array here (or a key/value map if we decide to change it per the last comment). This would return `enabledFeatures.includes( feature )` or just `!! enabledFeatures[ feature ]` accordingly 🙂